### PR TITLE
Fixes for new brig

### PR DESCRIPTION
**Pull request by spartanbobby**
# About the pull request

Fixes stuff about the new brig

Lights on windows
Lack of warning stripes under bulkheads
Lack of warning stripes under some doors
Fixes up some of the tiling to look better (hard to describe in the PR nanu knows about it)
Removes RNG loot spawner that could spawn thermals in the brig
Changes some 1x1 area
Changes the location of some of the semiotics 
Makes the north and south brig maint area symmetrical again
Fixes layering issues
Rewalls the CMPs office so that the outer brig hallways are symmetrical
Changes Brig Phone categories to "MP Dept."
fixes incorrect brig lockers
fixes incorrect door and button IDs
fixes incorrect door names
fixes incorrect pipe placement

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: SpartanBobby, LCMS1
maptweak: Numerous Fixes for new brig
/:cl:

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		"**/.pnp.*": true
 	},
 	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": true
```
Comment (fira) [inline .vscode/settings.json]: delet dis

**Comment by BadAtThisGame302** [comment]
Should probably fix that lil corner there too, it's one down.

image

**Comment by realforest2001** [comment]
Not sure if you picked it up, but the warden's office phone is sharing the name of the CMPs office phone

**Review by Nanu308 — APPROVED** [review]
fixes we need

**Pull request merged**